src/style.css

```diff
@@ -1,3 +1,87 @@
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(https://fonts.google.com/specimen/Roboto);
+}
+
+html{
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
+  background-color: #a09595;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  background-color: #fff;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  display: block;
+}
+
+.logo_image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav_list {
+  position: relative;
+  text-decoration: none;
+  color: black;
+  font-size: 12px;
+  text-transform: uppercase;
+  font-weight: 500;
+  line-height: 60px;
+  display: flex;
+  align-items: center;
+}
+
+.nav_item {
+  font-weight: 500;
+}
+
+li {
+  list-style-type: none;
```

Comment:
Avoid styling elements using tag selectors, as indicated in the checklist. This can lead to unintended side effects in your CSS when the HTML structure changes. Instead, style your elements using classes. In this case, you should add a class to your `li` elements and use that for styling.